pkgs/games/openmw/default.nix

```diff
@@ -3,46 +3,52 @@
   stdenv,
   fetchFromGitLab,
   fetchpatch,
-  cmake,
-  pkg-config,
-  wrapQtAppsHook,
+
   SDL2,
   boost,
   bullet,
-  # Please unpin this on the next OpenMW release.
-  ffmpeg_6,
+  cmake,
+  ffmpeg,
   libXt,
   luajit,
   lz4,
   mygui,
   openal,
   openscenegraph,
+  pkg-config,
+  qttools,
   recastnavigation,
   unshield,
+  wrapQtAppsHook,
   yaml-cpp,
+
+  withGL ? "GLVND",
```

Comment:
`withXXX` are *typically* used for boolean flags. I suggest something like `GLPreference`